bat/ASM.BAT

```diff
@@ -1,28 +1,41 @@
-@echo off
+@if /I "%1" == "tron" (echo on) else (echo off)
```

Comment:
What happens if user runs "bat/ASM.BAT" with no operands?

Reply:
you will get the error message define in lines 14-17